website/docs/release.md

```diff
@@ -4,18 +4,73 @@ title: Release Process
 
 ## Overview
 
-The release process for Copacetic uses [GoReleaser](https://goreleaser.com/). 
+This document describes the release management process for the Copacetic project, including release versioning, supported releases, upgrade guidelines, and operational details for cutting and publishing releases.
 
-Once you are ready to cut a new release, checkout the release branch and tag it with the respective version.
+## Legend
 
-	```
-	git checkout <BRANCH NAME>
-	git pull origin <BRANCH NAME>
-	git tag -a <NEW VERSION> -m '<NEW VERSION>'
-	git push origin <NEW VERSION>
-	```
+- **X.Y.Z** refers to the version (git tag) of Copacetic that is released. This is the version of Copacetic and any associated artifacts.
+- **Breaking changes** refer to schema changes, flag changes, or behavioral changes in Copacetic that may require a clean installation during upgrade and may introduce changes that could break backward compatibility.
+- **Milestone** should be designed to encapsulate feature sets that fit within a 3-month release cycle, including testing gates. GitHub's milestones are used by maintainers to manage each release. PRs and Issues for each release should be tracked as part of a corresponding milestone.
+- **Patch releases** refer to applicable fixes, including security fixes, that may be backported to supported releases, depending on severity and feasibility.
+- **Test gates** should include soak tests and upgrade tests from the last minor version.
```

Comment:
we don't currently have soak tests, maybe we can change this to include relevant integration and unit tests to the release?